File: CyFile-Android/app/src/main/java/at/tugraz/tc/cyfile/logging/impl/NoOpLogger.java
package at.tugraz.tc.cyfile.logging.impl;

import at.tugraz.tc.cyfile.logging.CyFileLogger;

public class NoOpLogger implements CyFileLogger {
    @Override
    public void d(String tag, String msg) {
    }

    @Override
    public void e(String tag, String msg) {
    }

    @Override
    public void i(String tag, String msg) {

    }

    @Override
    public void w(String tag, String msg) {

    }
}

File: CyFile-Android/app/src/main/java/at/tugraz/tc/cyfile/logging/impl/NoOpLogger.kt
package at.tugraz.tc.cyfile.logging.impl

import at.tugraz.tc.cyfile.logging.CyFileLogger

class NoOpLogger : CyFileLogger {
    override fun d(tag: String, msg: String) {}

    override fun e(tag: String, msg: String) {}

    override fun i(tag: String, msg: String) {

    }

    override fun w(tag: String, msg: String) {

    }
}
